### ISSUE TITLE
Some images are missing from colorscheme page

### ISSUE THREAD

**Issue by TahaTesser** [rydmike/rydmike.github.io#3]
Source: https://rydmike.com/colorscheme

### Preview

![Screen Shot 2021-11-19 at 16 34 27](https://user-images.githubusercontent.com/48603081/142640162-86181a99-d735-4573-a19a-291451876928.png)

**Comment by rydmike** [comment]
Thanks @TahaTesser, yes I noticed it yesterday too and fixed it then. It happened because as a part of releasing FlexColorScheme V4 I had to restructure where I keep my my assets for its docs quite a bit, and I had forgotten to update the links in this doc,

Thanks for letting me know. I still need to update this page to be based on V4 as well.

I plan to move the docs for FlexColorScheme to a separate docs site and only keep it minimal in the packages repo readme and on this page as well. It will be easier to maintain and read that way going forward.

**Comment by TahaTesser** [comment]
Cool website, really useful, keep it up

**Comment by rydmike** [comment]
Thx @TahaTesser this is just a simple collection of markdown pages that GitHub pages sites converts to static html. Been to lazy to make a fancy home page, besides not a big fan if html+css+js mess :)

I just wrote down a few things, mostly to remember and find them myself. Plus I host some Flutter Web apps here as you noticed, it is just so easy to deploy them here with GitHub actions.

But happy if others find the content useful too. I noticed my lint rules comparison sheet is at least popular. I should update it too, at minimum check changes for latest versions.

Been thinking of experimenting with making a Flutter site for my web content, just to prove it can be done. Saw some people doing it already and it looked pretty ok.
